DS2S META/Utils/Offsets/DS2SOffsetsV103.cs

```diff
@@ -17,12 +17,19 @@ public DS2SOffsetsV103()
             BIKP1Skip_Val1 = new int[] { 0x70, 0x20, 0x18, 0xe34 };
             BIKP1Skip_Val2 = new int[] { 0x70, 0x20, 0x18, 0xd52 };
 
-            
+
+            DisablePartyWalkTimer = new int[] { 0x18, 0xD70, 0x10, 0x0};
+            //GrabParamID = new int[] { 0x18, 0xD70, 0x10, 0x8 };
```

Comment:
What was the intention behind this piece of code?
If it could be useful for something else maybe leave a comment, otherwise I think we should remove it so it's not distracting for people trying to understand the source.

Reply:
I completly forgot about the existance of this comment, it used to be useful but i ended up using the base parameter and get the offset this way, so it can be removed